ci/install.sh

```diff
@@ -50,3 +50,6 @@ else
   gcc -m32 -o "$TREE_SITTER_DIR/bin/c.so" -shared parser.c -I.
 fi
 test -f "$TREE_SITTER_DIR/bin/c.so"
+
+sudo cpanm -n Neovim::Ext || cat "$HOME/.cpanm/build.log"
```

Comment:
This fails when running `Neovim::Ext` tests because it can't download nvim v0.4.3.

Reply:
It should only download nvim if it is not on the path, and when running tests. ```-n``` here should skip the tests?

Reply:
The issue is it can't download nvim when cpanm verified that the required Http perl modules are installed. Neovim::Ext dependencies don't fail their tests so `-n` is only for `Neovim::Ext` tests.

Reply:
```-n``` should apply to the distribution you're installing and all of its dependencies, if it doesn't, then I believe its a bug in ```App::cpanminus```. ```Neovim::Ext``` downloads neovim to verify that the API actually works. Not sure how the HTTP perl modules is in play here?

Reply:
https://github.com/jacquesg/p5-Neovim-Ext/commit/82565d72a8659ebca9196cd672e44f6c7f436b01#diff-8424792320c9483850c5842f19129c9bR89-R94

Reply:
I don't get what your point is? Like I said above, yes, ```Neovim::Ext``` downloads nvim to run its test if its not on the ```PATH```. However, running ```cpanm``` with the ```-n```/```--notest``` switch will skip tests and the download of neovim all together.